addons/xbmc.addon/metadata.xsd

```diff
@@ -3,20 +3,22 @@
 <xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
   <xs:element name="extension">
     <xs:complexType>
-      <xs:sequence>
+      <xs:choice maxOccurs="unbounded">
         <xs:element name="summary" type="translatedString" minOccurs="1" maxOccurs="unbounded"/>
         <xs:element name="description" type="translatedString" minOccurs="0" maxOccurs="unbounded"/>
-        <xs:element name="disclaimer" type="translatedString" minOccurs="0"/>
+        <xs:element name="disclaimer" type="translatedString" minOccurs="0" maxOccurs="unbounded"/>
         <xs:element name="platform" type="platformList" minOccurs="0"/>
-        <xs:element name="supportedcontent" type="contentList" minOccurs="0"/>
-        <xs:element name="language" type="xs:string"  minOccurs="0"/>
-        <xs:element name="license" type="xs:string"  minOccurs="0"/>
-        <xs:element name="forum" type="xs:string" minOccurs="0"/>
-        <xs:element name="website" type="xs:string" minOccurs="0"/>
-        <xs:element name="source" type="xs:string" minOccurs="0"/>
-        <xs:element name="email" type="xs:string" minOccurs="0"/>
-        <xs:element name="broken" type="xs:string" minOccurs="0"/>
-      </xs:sequence>
+        <xs:element name="language" type="xs:string"  minOccurs="0" maxOccurs="1"/>
+        <xs:element name="license" type="xs:string"  minOccurs="1" maxOccurs="1" />
```

Comment:
Is there any harm in an addon failing being checked against this xsd? It's just here for reference right, otherwise, I'm pretty sure this line will cause quiet some addons to fail.

Reply:
Not sure I understand what you mean. This is not used (it's just reference) but since we require addon developers to include a license with the addon in our submission guidelines (https://kodi.wiki/view/Add-on_rules) and have been suggesting them to define the `license` element (wiki and website) I decided to make it mandatory. Addons will not fail to install if they have no license file though. Maybe it's better to leave it with `minOccurs="0"` and create a new check in the addon-checker if necessary?

Reply:
No, it's fine that's what I was asking.

Reply:
The question is should the schemas in this repo describe our addon submission rules or what Kodi can handle?

Reply:
Personally in the case of the license element I am okay with both. In other cases I'd prefer the schema to describe what kodi can actually handle so..I'll change it.